doc/api/brotli.md

```diff
@@ -248,47 +256,57 @@ Parameters.
 * `brotli.constants.BROTLI_DECODER_PARAM_DISABLE_RING_BUFFER_REALLOCATION`
 * `brotli.constants.BROTLI_DECODER_PARAM_LARGE_WINDOW`
 
-## Class: Options
+## Options Object
+<!-- YAML
+added: REPLACEME
+-->
+
 <!--type=misc-->
 
 Each class takes an `options` object. All options are optional.
 
-Note that some options are only relevant when compressing, and are
-ignored by the decompression classes.
+Note that some options are only relevant when compressing or decompressing,
+and are simply ignored.
```

Comment:
I am not sure, but it seems a bit incomplete or confusing now. Maybe something like "and are simply ignored when inappropriate."?

Reply:
Something like "and invalid options are simply ignored" should also work.